updating README with new bolt instructions

MirandaStreeter opened the pull request.

Commit:
updating README with new bolt instructions

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/puppetlabs/ca_extend?pullRequest=8) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/puppetlabs/ca_extend?pullRequest=8) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: m0dular<br/>:x: student14<br/><hr/>**student14** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/puppetlabs/ca_extend?pullRequest=8) it.</sub>

m0dular committed:
Update README to use Bolt Projects

Prior to this commit, the instructions used a deprecated method of
installing modules using a `Boltdir`.  This commit updates the
documentation to use a Project and provides instructions for installing
behind a proxy.

m0dular commented:
Created #9 so the cla bot is happy.
